src/transforms/transform-deployment.ts

```diff
@@ -203,8 +203,8 @@ export const mapState = (state: string | undefined): string => {
 	}
 };
 
-const matchesEnvironment = (environment: string, globPatterns: string[] = []): boolean => {
```

Comment:
In existing code, if `null` is passed in, the `globPatterns` will not be replaced by `[]`. 
ps: I am a bit worry about the rest of our code (where default value is used) as well, but will leave it for next time.